lib/ansible/modules/cloud/azure/azure_rm_sqlserver.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sqlserver
+version_added: "2.5"
+short_description: Manage SQL Server instance
+description:
+    - Create, update and delete instance of SQL Server
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    name:
+        description:
+            - The name of the server.
+        required: True
+    tags:
```

Comment:
I'm pretty sure you don't need the tags as you are extending the `azure_tags` documentation below.